interface/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+//! Instructions and constructors for the System program.
+//!
+//! The System program is responsible for the creation of accounts and [nonce
+//! accounts][na]. It is responsible for transferring lamports from accounts
+//! owned by the System program, including typical user wallet accounts.
+//!
+//! [na]: https://docs.solanalabs.com/implemented-proposals/durable-tx-nonces
+//!
+//! Account creation typically involves three steps: [`allocate`] space,
+//! [`transfer`] lamports for rent, [`assign`] to its owning program. The
+//! [`create_account`] function does all three at once. All new accounts must
+//! contain enough lamports to be [rent exempt], or else the creation
+//! instruction will fail.
+//!
+//! [rent exempt]: https://solana.com/docs/core/accounts#rent-exemption
+//!
+//! The accounts created by the System program can either be user-controlled,
+//! where the secret keys are held outside the blockchain,
+//! or they can be [program derived addresses][pda],
+//! where write access to accounts is granted by an owning program.
+//!
+//! [pda]: solana_pubkey::Pubkey::find_program_address
+//!
+//! Most of the functions in this module construct an [`Instruction`], that must
+//! be submitted to the runtime for execution, either via RPC, typically with
+//! [`RpcClient`], or through [cross-program invocation][cpi].
+//!
+//! When invoking through CPI, the [`invoke`] or [`invoke_signed`] instruction
+//! requires all account references to be provided explicitly as [`AccountInfo`]
+//! values. The account references required are specified in the documentation
+//! for the [`SystemInstruction`] variants for each System program instruction,
+//! and these variants are linked from the documentation for their constructors.
+//!
+//! [`RpcClient`]: https://docs.rs/solana-client/latest/solana_client/rpc_client/struct.RpcClient.html
+//! [cpi]: solana_program::program
+//! [`invoke`]: solana_program::program::invoke
+//! [`invoke_signed`]: solana_program::program::invoke_signed
+//! [`AccountInfo`]: solana_account_info::AccountInfo
+#![cfg_attr(feature = "frozen-abi", feature(min_specialization))]
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+
+pub mod error;
+pub mod instruction;
+#[cfg(target_arch = "wasm32")]
+mod wasm;
+
+use solana_pubkey::Pubkey;
+
+// inline some constants to avoid dependencies
+const RECENT_BLOCKHASHES_ID: Pubkey =
+    Pubkey::from_str_const("SysvarRecentB1ockHashes11111111111111111111");
```

Comment:
Can you add a static assertion that it's the same as `solana_program::sysvar::recent_blockhashes::id`?

Reply:
Maybe I'm missing something -- did you add this test?